packages/terra-clinical-item-view/tests/wdio/item-view-spec.js

```diff
@@ -0,0 +1,46 @@
+/* global before, browser, Terra */
+
+const viewports = Terra.viewports('tiny', 'small', 'medium', 'large', 'huge', 'enormous');
+const selector = '#root';
+
+describe('Item View', () => {
```

Comment:
Can we update these to screenshots to grab each example on the page instead of a screenshot of the entire visible viewport?  For example, comment display should grab separate images for default and truncated